resources/build/build-utils.sh

```diff
@@ -141,12 +141,41 @@ function printVersionUtilsDebug() {
     echo "VERSION_WITH_TAG: $VERSION_WITH_TAG"
 }
 
+function findShouldSentryRelease() {
+    # Default, in case $VERSION_ENVIRONMENT is improperly specified.
+    UPLOAD_SENTRY=false
+
+    # Dynamically configure whether or not to upload Sentry symbols.
+    case $VERSION_ENVIRONMENT in
+    # Actual release tiers
+    alpha)
+        UPLOAD_SENTRY=true
+        ;;
+    beta)
+        UPLOAD_SENTRY=true
+        ;;
+    stable)
+        UPLOAD_SENTRY=true
+        ;;
```

Comment:
```suggestion
    alpha | beta | stable)
        UPLOAD_SENTRY=true
        ;;
```

This isn't essential but maybe cleaner to read?

Reply:
Thanks; couldn't remember how to do that at the time in shell-script.